src/applications/auth/containers/AuthApp.jsx

```diff
@@ -133,10 +133,9 @@ export class AuthApp extends React.Component {
     const returnUrl = sessionStorage.getItem(authnSettings.RETURN_URL) || '';
     sessionStorage.removeItem(authnSettings.RETURN_URL);
 
-    const postAuthUrl =
-      returnUrl.includes('?next=') && !environment.isProduction()
-        ? appendQuery(returnUrl, 'postLogin=true')
-        : returnUrl;
+    const postAuthUrl = !environment.isProduction()
+      ? appendQuery(returnUrl, 'postLogin=true')
+      : returnUrl;
 
     const redirectUrl =
       (!returnUrl.match(REDIRECT_IGNORE_PATTERN) && postAuthUrl) || '/';
```

Comment:
I'm assuming it's fine to not include if we just redirect to the home page?

Reply:
Yeah it should be. I think they are only having issues when redirects happen from a page that requires logging in.